kurtosis-module/impl/participant_network/prelaunch_data_generator/el_cl_generation_test.go

```diff
@@ -6,33 +6,33 @@ import (
 	"github.com/kurtosis-tech/eth2-merge-kurtosis-module/kurtosis-module/impl/module_io"
 	"github.com/kurtosis-tech/eth2-merge-kurtosis-module/kurtosis-module/impl/participant_network/prelaunch_data_generator/cl_genesis"
 	"github.com/kurtosis-tech/eth2-merge-kurtosis-module/kurtosis-module/impl/participant_network/prelaunch_data_generator/el_genesis"
-	"github.com/kurtosis-tech/eth2-merge-kurtosis-module/kurtosis-module/impl/static_files"
 	"github.com/kurtosis-tech/kurtosis-core-api-lib/api/golang/lib/enclaves"
 	"github.com/kurtosis-tech/kurtosis-engine-api-lib/api/golang/lib/kurtosis_context"
 	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/require"
+	"io/ioutil"
 	"os"
 	"path"
 	"testing"
 	"time"
 )
 
 const (
-	runKurtosisTestsEnvVar	= "RUN_KURTOSIS_TESTS"
+	runKurtosisTestsEnvVar = "RUN_KURTOSIS_TESTS"
 
-	enclaveIdPrefix = "test-prelaunch-genesis-generation-"
+	enclaveIdPrefix       = "test-prelaunch-genesis-generation-"
 	isPartitioningEnabled = false
 
 	// Relative to the directory this file is inside
-	staticFilesRelDirpath = "../../static_files"
+	staticFilesRelDirpath = "../../../static_files"
```

Comment:
the previous path here was wrong even though the test itself never ran on CI or locally due to the skip